clusterloader2/testing/node-throughput/windows_override.yaml

```diff
@@ -0,0 +1,3 @@
+POD_COUNT: 80
+CONTAINER_IMAGE: gcr.io/gke-release/pause-win:1.0.0
+POD_STARTUP_LATENCY_THRESHOLD: 20m
```

Comment:
This seems to be a magnitude higher than the default (5s) - why?

Reply:
For experimenting with widnwos nodes it makes sense to me.

Reply:
Yeah, it was explained in the PR description. Looks good to me as well, although wouldn't hurt to add a TODO / comment here.

Reply:
I must have missed that. Make sense.